swupd/delta.go

```diff
@@ -67,15 +67,15 @@ func createDeltasFromManifests(c *config, oldManifest, newManifest *Manifest) ([
 		return []Delta{}, nil
 	}
 
-	// TODO: Unify these maxRoutines constants, get a better default (maybe NumCPUs)
-	// and make it configurable.
-	const maxRoutines = 3
+	if numWorkers < 1 {
```

Comment:
If you change the default to 1 instead of 0 you don't need this check anymore.

Reply:
Not sure how can I avoid this check. People can still pass 0 to the function when calling it.